### PR TITLE
automation: Fix Active Scan Policy job Rule add/modify Dialog sizing/display

### PR DESCRIPTION
**Pull request by kingthorin**
Prevent the dialog(s) from being initially displayed smaller than their contents.

### CODE REVIEW

**Comment by thc202** [comment]
I don't see any problem/overlapping of fields here, though pack will not hurt anyway, better with default spacing.
There's a typo in the commit message.

File: addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AddAscanRuleDialog.java [hunk 1]
```diff
@@ -108,6 +108,7 @@ public AddAscanRuleDialog(
         this.addComboField(STRENGTH_PARAM, allstrengths, strengthName);
 
         this.addPadding();
+        pack();
```
Comment (thc202) [inline addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AddAscanRuleDialog.java:111]: Might worth leaving a comment in the constructor saying that the dimension being passed will not make any difference.

**Comment by kingthorin** [comment]
The bottom was always too high for me, so things were partially blocked.

File: addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AddAscanRuleDialog.java (hunk 1 above)
Comment (kingthorin) [reply addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AddAscanRuleDialog.java:111]: Should I just remove the dimension?

Comment (kingthorin) [reply addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AddAscanRuleDialog.java:111]: Disregard, all of the constructors expect one.

**Comment by kingthorin** [comment]
Tweaked

File: addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AddAscanRuleDialog.java (hunk 1 above)
Comment (thc202) [reply addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AddAscanRuleDialog.java:112]: That would lead to NPE.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by psiinon** [comment]
Logo
**Checkmarx One – Scan Summary & Details** – [af4bd771\-9509\-4fcb\-bb86\-5d4c8789795d](https://eu.ast.checkmarx.net/projects/0da811fa-6373-42b1-bbb6-d0026285b136/scans?id=af4bd771-9509-4fcb-bb86-5d4c8789795d&branch=kingthorin%2Fauto-rule-dialogs)
### No New Or Fixed Issues Found

**Comment by thc202** [comment]
This has conflicts.

**Comment by kingthorin** [comment]
Deconflicted

**Review by psiinon — APPROVED** [review]

**Pull request merged**